providers/bitbucket.js

```diff
@@ -168,10 +168,52 @@ class BitBucket extends BaseProvider {
             icon_url: this.body.actor.links.avatar.href,
             url: this.baseLink + this.body.actor.username
         };
+
+        let changes = [];
+        if (typeof this.body.changes !== "undefined") {
+            const states = ['old', 'new'];
+
+            ['Assignee', 'Responsible'].forEach((label) => {
+                const actor = this.body.changes[label.toLowerCase()],
+                    actorNames = {};
+
+                if (actor === 'undefined') {
+                    return;
+                }
+
+                states.forEach((state) => {
+                    if (typeof actor[state] === 'object' && typeof actor[state].username === 'string') {
+                        actorNames[state] = actor[state].username;
+                    }
+                });
+
+                if (!Object.keys(actorNames).length) {
+                    return;
+                }
+
+                states.forEach((state) => {
+                    if (typeof actorNames[state] === 'undefined') {
+                        actorNames[state] = '***null***';
+                    }
+                });
+
+                changes.push("**" + label + ":** \"" + actorNames.old + "\" -> \"" + actorNames.new + "\"");
+            });
+
+            ['Priority', 'Status', 'Content', 'Kind'].forEach((label) => {
+                const property = this.body.changes[label.toLowerCase()];
+
+                if (typeof property !== "undefined") {
+                    changes.push("**" + label + ":** \"" + property.old + "\" -> \"" + property.new + "\"");
```

Comment:
I just loaded this up locally and tested this out. I have two design suggestions. The first is to use graves ` instead of double quotes for the asignee and priority values. The second is that it might be better to omit the "old content" since descriptions of issues can be large, and webhook users might not even care what the old value was, they just want to see what it has been updated to. Another thing with the content is that large issue descriptions may exceed the character count for discord embeds. It might be benefitial to use a character check. This is done in several other providers, but I'll link you to the trello implementation of it https://github.com/Commit451/skyhook/blob/master/providers/trello.js#L680

Reply:
I agree with all these changes.
* I wanted to use template literals but wanted to stay consistent with the double quotes used throughout the project
* Agreed, no "old" description
* I didn't know about the 2048 character limit. I'll have to reorganize the different updates properties to have the most important at the top, and put the "Content" property (the description) at the bottom, it being the most likely culprit of a long message

I'll implement these changes today if I have time, or over the weekend.